tslint.json

```diff
@@ -0,0 +1,20 @@
+{
+  "defaultSeverity": "error",
+  "extends": [
+    "tslint:recommended"
+  ],
+  "jsRules": {},
+  "rules": {
+    "quotemark": [true, "single", "jsx-double", "avoid-template"],
```

Comment:
I put most of these rules in here to minimize the changes to the files I converted to TS.